src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/dec/BrotliStream.Decompress.cs

```diff
@@ -27,6 +39,12 @@ public override int ReadByte()
             return numRead != 0 ? b : -1;
         }
 
+        /// <summary>Reads a sequence of bytes from the current Brotli stream to a byte span and advances the position within the Brotli stream by the number of bytes read.</summary>
+        /// <param name="buffer">A region of memory. When this method returns, the contents of this region are replaced by the bytes read from the current source.</param>
+        /// <returns>The total number of bytes read into the buffer. This can be less than the number of bytes allocated in the buffer if that many bytes are not currently available, or zero (0) if the end of the stream has been reached.</returns>
+        /// <remarks>Use the <see cref="System.IO.Compression.BrotliStream.CanRead" /> property to determine whether the current instance supports reading. Use the <see cref="System.IO.Compression.BrotliStream.ReadAsync" /> method to read asynchronously from the current stream.
```

Comment:
This cref is ambiguous because there are two methods with that name, so the parameters need to be indicated, or I need to confirm there's a way to refer to all overloads at once (In MS Docs, we use `%2A` as the suffix for that purpose):
```cs
public override Task<int> ReadAsync(byte[] buffer, int offset, int count, CancellationToken cancellationToken)
public override ValueTask<int> ReadAsync(Memory<byte> buffer, CancellationToken cancellationToken = default(CancellationToken))
```

Reply:
There's a request to add this feature in the dotnet/csharplang repo: https://github.com/dotnet/csharplang/issues/320#issuecomment-769473469

I mentioned this issue.